api.php

```diff
@@ -400,24 +400,42 @@ function listOutOfDateMods($modsWithVersions) {
 	$combinedParams = implode(",", array_fill(0, count($combined), "?"));
 	$modidParams = implode(",", array_fill(0, count($modids), "?"));
 
-	// Since this query's join on `latestrelease` limits the output to only the latest release for each mod,
-	// We can filter out releases where the incoming version param is the same as the latest release
-	// This means we can use the query itself to fetch only mods which are not on the latest version :)
 	$outOfDateMods = array();
 	$latestReleases = $con->getAll("
+		with latestrelease as (
+			select 
+				`release`.modid,
+				`release`.releaseid,
+				`release`.modidstr,
+				`release`.modversion,
+				`release`.created,
+				`release`.assetid,
+				max(`release`.modversion) 

```

Comment:
pretty sure max() interprets the version as a decimal number, which is also not correct. I also doubt it can correctly interpret pre.5 as a lower version number than rc.1.

You will need to retrieve all releases and use the same sorting algorithm as in show-mod.php, line 106

Reply:
Good catch! I've made a refactor of the endpoint to retrieve all releases and compare them with the same logic.

Reply:
thanks